src/main/python/main/ayab/engine/communication_ip.py

```
@@ -0,0 +1,215 @@
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with AYAB.  If not, see <http://www.gnu.org/licenses/>.
+#
+#    Copyright 2024 Marcus Hoose (eKnitter.com)
+"""Handles the IP communication protocol.
+
+This module handles IP communication, currently works in a synchronous way.
+"""
+
+from .communication import *
```

Ruff

`from .communication import *` used; unable to detect undefined names

(F403)